sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample4_Logging.cs

```diff
@@ -24,8 +24,7 @@ public void Logging()
             var client = new ConfigurationClient(connectionString);
 
             // Setup a listener to monitor logged events.
-            var listener = new ConsoleEventListener();
-            listener.EnableEvents(EventLevel.LogAlways);
+            using AzureEventSourceListener listener = AzureEventSourceListener.CreateConsoleLogger(EventLevel.LogAlways);
```

Comment:
>using AzureEventSourceListener listener = AzureEventSourceListener.CreateConsoleLogger(EventLevel.LogAlways); [](start = 12, length = 109)

Should the sample also show overriding the AzureEventSourceListener as well as log to console?

Reply:
Will do